File: ejercicios parte 1/2)estructuras condicionales/11determinar par/determinar_par.java
import java.util.Scanner;

public class determinar_par {
    public static void main(String[] args) {
        try (Scanner scan = new Scanner(System.in)) {
            int num;

            System.out.print("Ingrese un número: ");
            num = scan.nextInt();
            if(num%2==0){
                System.out.println("Su número es par");
            }else{
                System.out.println("Su número es impar");
            }
        }
    }
}

File: ejercicios parte 1/2)estructuras condicionales/11determinar par/determinar_par.cpp
#include <iostream>
using namespace std;
int main(){
    int num;
    cout<<"Ingrese un número: ";
    cin>>num;

    if(num%2==0){
        cout<<"Es un número par"<<endl;
    }else{
        cout<<"Es un número impar"<<endl;
    }
}